docs/sql-data-sources-hive-tables.md

```diff
@@ -119,15 +119,15 @@ One of the most important pieces of Spark SQL's Hive support is interaction with
 which enables Spark SQL to access metadata of Hive tables. Starting from Spark 1.4.0, a single binary
 build of Spark SQL can be used to query different versions of Hive metastores, using the configuration described below.
 Note that independent of the version of Hive that is being used to talk to the metastore, internally Spark SQL
-will compile against Hive 1.2.1 and use those classes for internal execution (serdes, UDFs, UDAFs, etc).
+will compile against Hive 2.3.6 and use those classes for internal execution (serdes, UDFs, UDAFs, etc).
```

Comment:
How much do we want to commit to a particular version, vs just saying 2.3? I don't feel strongly about it.

Reply:
+1 for 2.3.

Reply:
This seems to become `built-in Hive` eventually. +1 for that.